bin/yb-ctl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Let's still keep this as `#!/usr/bin/env python` and make sure it works with Python 2 and Python 3.